test/views/editLiability/disposeLiabilityHasBankDetailsSpec.scala

```diff
@@ -57,7 +57,6 @@ implicit val request = FakeRequest()
 
       Then("The date fields should have the correct titles")
       And("No data is populated")
-      assert(document.getElementById("hasBankDetails-id").text() === "Do you have a bank account where we could pay a refund? Yes No")
```

Comment:
Can you change the test rather than remove it. We need to test that something is there.